app/assets/javascripts/discourse/components/d-editor.js.es6

```diff
@@ -162,6 +168,11 @@ class Toolbar {
       g.buttons.push(createdButton);
     }
   }
+
+  getButtonLabel(labelKey, defaultLabel) {
```

Comment:
This is minor but it seems `getButtonLabel` never uses `this` so it doesn't need to be an object method. You can hoist it up outside of the object declaration and it should still work.